app/src/main/java/com/ataulm/artcollector/TemporaryHomeActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.ataulm.artcollector
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+
+/**
+ * This activity will be removed. It's necessary for the base feature to have a launcher activity.
+ * This will be `PaintingsActivity`, which will be consumed by this module (`app`) again, when I add another feature module.
+ */
+class TemporaryHomeActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        startActivity(Navigation.PAINTINGS.viewIntent())
+    }
+}
+
+enum class Navigation(private val semiQualifiedActivityName: String) {
+
+    PAINTINGS("paintings.ui.PaintingsActivity");
+
+    fun viewIntent(): Intent {
+        val componentName = "${BuildConfig.APPLICATION_ID}.$semiQualifiedActivityName"
+        return Intent(Intent.ACTION_VIEW)
+                .setClassName(BuildConfig.APPLICATION_ID, componentName)
```

Comment:
this is the "hack". I _guess_ we'll be able to get rid of it when we implement AppLinks, but when I tried using:

`startActivity(Intent("com.ataulm.artcollector.VIEW_PAINTINGS"))` 

(with the corresponding intent filter in the `PaintingsActivity` in `paintings/.../AndroidManifest`) it said it couldn't find the activity and would crash.

I'm not sure how this method makes the activity "findable".